### PR TITLE
16/04/24 merge branch

### PR DESCRIPTION
**Pull request by daniel-sussman**

### CODE REVIEW

File: app/models/company.rb
Comment (Ches-ctrl) [inline app/models/company.rb:1]: They should just be name and website url - the repetition is a holdover from the bootcamp

Comment (Ches-ctrl) [reply app/models/company.rb:1]: Need to add to README & ENV file

File: app/models/concerns/ats/ashbyhq/company_details.rb
Comment (Ches-ctrl) [inline app/models/concerns/ats/ashbyhq/company_details.rb:1]: How do these hashes work?

File: app/models/concerns/ats/devitjobs/job_details.rb
Comment (Ches-ctrl) [inline app/models/concerns/ats/devitjobs/job_details.rb:1]: To review with Dan

File: app/models/concerns/ats/greenhouse/fetch_company_jobs.rb
Comment (Ches-ctrl) [inline app/models/concerns/ats/greenhouse/fetch_company_jobs.rb:1]: We have get, get_response and get_json_data methods - do we need all of these?

File: app/models/concerns/ats/pinpointhq/company_details.rb
Comment (Ches-ctrl) [inline app/models/concerns/ats/pinpointhq/company_details.rb:1]: jobglob has the scraper functionality as a a separate class that then imports the specifics of the ATS system in - we may wish to do the same in due course

File: app/models/concerns/ats_url_identifiers.rb
Comment (Ches-ctrl) [inline app/models/concerns/ats_url_identifiers.rb:1]: Don't fully understand what this is doing at the moment - let's discuss where it best sits

File: app/models/concerns/relevant.rb
Comment (Ches-ctrl) [inline app/models/concerns/relevant.rb:1]: Nice, think this living separately works

File: app/models/job.rb
Comment (Ches-ctrl) [inline app/models/job.rb:1]: Nice

File: app/services/company_industry_service.rb
Comment (Ches-ctrl) [inline app/services/company_industry_service.rb:1]: To discuss whether we want this dependency

File: app/services/import_companies_from_list.rb
Comment (Ches-ctrl) [inline app/services/import_companies_from_list.rb:1]: To delete

File: app/services/job_url.rb
Comment (Ches-ctrl) [inline app/services/job_url.rb:1]: To delete

**Review by Ches-ctrl — APPROVED** [review]

File: app/models/concerns/ats/greenhouse/fetch_company_jobs.rb
Comment (daniel-sussman) [reply app/models/concerns/ats/greenhouse/fetch_company_jobs.rb:1]: They all do separate things. get_json_data calls get, so we're still DRY.

**Pull request merged**